src/kontrol/__main__.py

```diff
@@ -305,9 +305,11 @@ def exec_prove(
     for proof in results:
         if proof.passed:
             print(f'PROOF PASSED: {proof.id}')
+            print(f'time: {proof.exec_time//3600}h {(proof.exec_time%3600)//60}m {proof.exec_time%60}s')
```

Comment:
```suggestion
            print(f'time: {proof.exec_time // 3600}h {(proof.exec_time % 3600) // 60}m {proof.exec_time % 60}s')
```

Reply:
Also, since the same statement is repeated below, maybe it makes sense to factor it out in a separate function. This string
```
f'{proof.exec_time//3600}h {(proof.exec_time%3600)//60}m {proof.exec_time%60}s'
```
could also be a property of the proof object (e.g., `formatted_time`, or something like that), which would require another PR in pyk.

Reply:
Good idea! I just opened this [PR](https://github.com/runtimeverification/pyk/pull/941) in pyk.